lib/app.go

```diff
@@ -139,7 +135,7 @@ func appState(app *App, pod *pkgPod.Pod) error {
 	}
 
 	app.State = AppStateCreated
-	createdAt := fi.ModTime().UnixNano()
```

Comment:
fixed in https://github.com/coreos/rkt/pull/3318/commits/2f5de7f341ca626a140ee9a43f06254fb53cb301